packages/react/kibana_context/common/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
```

Comment:
The backport has to be manual because some of these bazel files where added in main before in https://github.com/elastic/kibana/pull/199748 but weren't backported to 8.x